src/index.css

```diff
@@ -0,0 +1,13 @@
+/* define colors */
+:root {
+  --graasp-primary: #5050d2;
+}
+
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
```

Comment:
@spaenleh @pyphilia, is this what we use in UI?

Reply:
I will remove this, as I am using the `CssBaseline` component from MUI that sets a base style automatically.